File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroFomites.java
package com.rinconadadelsur.sistemas.bioseguridad.Entidades;

public class eRegistroFomites {

    private String _codigo;
    private String _depredador;
    private String _tipo;
    private String _observacion;

    public eRegistroFomites(){

    }

    public eRegistroFomites(String _codigo, String _depredador, String _tipo, String _observacion) {
        this._codigo = _codigo;
        this._depredador = _depredador;
        this._tipo = _tipo;
        this._observacion = _observacion;
    }

    public String get_codigo() {
        return _codigo;
    }

    public void set_codigo(String _codigo) {
        this._codigo = _codigo;
    }

    public String get_depredador() {
        return _depredador;
    }

    public void set_depredador(String _depredador) {
        this._depredador = _depredador;
    }

    public String get_tipo() {
        return _tipo;
    }

    public void set_tipo(String _tipo) {
        this._tipo = _tipo;
    }

    public String get_observacion() {
        return _observacion;
    }

    public void set_observacion(String _observacion) {
        this._observacion = _observacion;
    }

    @Override
    public String toString() {
        return _observacion;
    }
}

File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroFomites.kt
package com.rinconadadelsur.sistemas.bioseguridad.Entidades

class eRegistroFomites {
    private var _codigo: String? = null
    private var _depredador: String? = null
    private var _tipo: String? = null
    private var _observacion: String? = null

    constructor()

    constructor(_codigo: String?, _depredador: String?, _tipo: String?, _observacion: String) {
        this._codigo = _codigo
        this._depredador = _depredador
        this._tipo = _tipo
        this._observacion = _observacion
    }

    fun get_codigo(): String? {
        return _codigo
    }

    fun set_codigo(_codigo: String?) {
        this._codigo = _codigo
    }

    fun get_depredador(): String? {
        return _depredador
    }

    fun set_depredador(_depredador: String?) {
        this._depredador = _depredador
    }

    fun get_tipo(): String? {
        return _tipo
    }

    fun set_tipo(_tipo: String?) {
        this._tipo = _tipo
    }

    fun get_observacion(): String {
        return _observacion!!
    }

    fun set_observacion(_observacion: String) {
        this._observacion = _observacion
    }

    override fun toString(): String {
        return _observacion!!
    }
}
